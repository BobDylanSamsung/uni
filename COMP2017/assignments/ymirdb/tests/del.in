set a 1 2 3
set b 4 5 6
set c 9 8 7 0
list entries
del c
list entries
del a
list entries
del b
list entries
bye
