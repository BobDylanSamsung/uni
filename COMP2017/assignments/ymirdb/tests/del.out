ok

ok

ok

c [9 8 7 0]
b [4 5 6]
a [1 2 3]

ok

b [4 5 6]
a [1 2 3]

ok

b [4 5 6]

ok

no entries

bye
